src/Cli/dotnet/commands/dotnet-sdk/check/SdkOutputWriter.cs

```diff
@@ -36,7 +36,7 @@ public void PrintSdkInfo()
             if (NewFeatureBandAvailable())
             {
                 _reporter.WriteLine();
-                _reporter.WriteLine(string.Format(LocalizableStrings.NewFeatureBandMessage, NewestFeatureBandAvailable()));
+                _reporter.WriteLine(string.Format(LocalizableStrings.NewFeatureBandMessage, NewestFeatureBandAvailable()));         // this is where a new feature band is advertised
```

Comment:
I would suggest moving this comment above, and maybe simplify the comment to something like "Advertise the new feature band". IMO comments should prescriptive and not descriptive as the former may cue someone debugging the code into incorrect assumptions